### ISSUE TITLE
Error in nvs for 2 worded usernames

### ISSUE THREAD

**Issue by vickylance** [jasongin/nvs#64]
Hi I used the latest stable version of nvs. But since the username is 2 words seperated by a space, I am getting an error.

```
C:\Users\Krithika : The term 'C:\Users\Krithika' is not recognized as the name of a cmdlet, function, script file, or
operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try
again.
At line:1 char:1
+ C:\Users\Krithika Senthil\AppData\Local\nvs\nvs.ps1 bootstrap
+ ~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\Krithika:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

```
So please let me know what change I have to do in the nvs.cmd file to make it work?

**Comment by jasongin** [comment]
Hi @vickylance,
Thanks for the bug report. I can reproduce this problem and I'll make a fix available soon.

**Comment by jasongin** [comment]
Fixed as https://github.com/jasongin/nvs/commit/d281197357889ba96ecb311f3b12ab0dfe900142

**Comment by jasongin** [comment]
I published a [v1.4.2 release](https://github.com/jasongin/nvs/releases/tag/v1.4.2) with this fix.

**Comment by vickylance** [comment]
Hi. @jasongin  Thanks for fixing the issue and publishing an update but I am facing an issue with the latest update.I am not able to install the package. I have raised a new issue regarding this.
